{"name": "登录用户名错误", "status": "passed", "attachments": [{"name": "log", "source": "7c6beaea-73fd-4098-800a-8c3d55f75b14-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "testcase_name", "value": "'登录用户名错误'"}, {"name": "testcase", "value": "{'name': '登录用户名错误', 'action': '/keyword/actions/login.yml', 'params': {'username': 18837774444}, 'validate': [{'type': 'page_contains', 'expect': '帐号密码登录'}]}"}], "start": 1634738073363, "stop": 1634738078255, "uuid": "8af41ca8-8620-4b6e-806d-dbff70ef4603", "historyId": "79e160cbe898ed661638acc90b58ae99", "testCaseId": "7b3c8701b03d8fb259b18b98c43b083a", "fullName": "keyword.keyword_run#test_keyword", "labels": [{"name": "parentSuite", "value": "keyword"}, {"name": "suite", "value": "keyword_run"}, {"name": "host", "value": "LAPTOP-93HH27QA"}, {"name": "thread", "value": "31280-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "keyword.keyword_run"}]}